packages/contentstack-audit/bin/dev

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env node
```

Comment:
can we use the latest method for the dev ?

Reply:
What do you mean by `latest method for the dev`? Can you rephrase it.?

Reply:
#!/usr/bin/env node
require('dotenv').config({ path: './.env' });
(async () => {
  const { execute } = require('@contentstack/cli-utilities');
  await execute({ type: 'cjs', development: true, dir: __dirname });
})();

we could use this instead